cpp/src/ray/address_helper.cc

```diff
@@ -0,0 +1,16 @@
+#include <dlfcn.h>
```

Comment:
Better move `cpp/src/ray/address_helper.cc` to `cpp/src/ray/api/util/address_helper.cc`.